notification/src/handler/notification/notification.handler.js

```diff
@@ -108,6 +108,7 @@ function getAddInterfaceInteractionUpdateAction (notification) {
     fields: {
       createdAt: new Date(),
       status: notification.NotificationRequestItem.eventCode,
+      type: 'NOTIFICATION',
```

Comment:
why not lowercase?

Reply:
also, you could add a check for the tests, right?

Reply:
Because in the field `status` one line above, it's also uppercase.

Reply:
I guess we should use the same idea with the extension module here, not a big change IMO.

Reply:
I did not touch the extension module as you're working on it. So I guess it's up to you to decide what to use here :)

Reply:
>  So I guess it's up to you to decide what to use here :) 

could you please change both of them to lowercase?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/233/commits/9654a8075e0da13b99106a3f7c53eea646332046

Reply:
Do you want to keep the status field on notification? What benefit do you see and why not name it eventCode as provided by the Adyen?

Reply:
In extension we also have status, so I would keep it to have a consistent format. Also it's quite useful to know which kind of notification is that from the first glance without needing to look in the JSON and decoding the status. I believe Hasan already talked to Adyen about the meaning of eventCode and it's the status of notification.